ml-agents-envs/mlagents/envs/base_env.py

```diff
@@ -18,12 +18,12 @@
 """
 
 from abc import ABC, abstractmethod
-from typing import List, NamedTuple, Tuple, Optional, Union, Dict, NewType
+from typing import List, NamedTuple, Tuple, Optional, Union, Dict
 import numpy as np
 from enum import Enum
 
-AgentId = NewType("AgentId", int)
-AgentGroup = NewType("AgentGroup", str)
+AgentId = int
```

Comment:
I think this was going to be more trouble that it was worth. This still aliases the types so that it's a little clearer than the raw int/str types.